src/backend/marsha/core/tests/__init__.py

```diff
@@ -1 +1,3 @@
 """Tests for the ``core`` app of the Marsha project."""
+# pylint: disable=wildcard-import,unused-wildcard-import
+from .utils import *  # noqa isort:skip
```

Comment:
I don't see the point of this. Importing from `tests.utils` is more explicit?

Reply:
I wasn't sure of this. Both seems ok for me.